src/Uno.UI.RemoteControl.Host/Uno.UI.RemoteControl.Host.csproj

```diff
@@ -35,6 +35,20 @@
 		</ProjectReference>
 	</ItemGroup>
 
+	<ItemGroup>
+		<Content Update="Config\*.json">
+			<CopyToOutputDirectory>Never</CopyToOutputDirectory>
+		</Content>
+		<Content Update="Config\global-net7.0.json" Condition="'$(TargetFramework)' == 'net7.0'">
+			<CopyToOutputDirectory>Always</CopyToOutputDirectory>
+			<TargetPath>global.json</TargetPath>
+		</Content>
+		<Content Update="Config\global-net8.0.json" Condition="'$(TargetFramework)' == 'net8.0'">
+			<CopyToOutputDirectory>Always</CopyToOutputDirectory>
+			<TargetPath>global.json</TargetPath>
+		</Content>
+	</ItemGroup>
```

Comment:
These files are part of https://github.com/unoplatform/uno/blob/master/build/nuget/Uno.WinUI.RemoteControl.nuspec and may need to be modified there as well.

Reply:
The `RemoteControl.nuspec` is for the deprecated package, but I validated it's correct in the DevServer.nupsec : https://github.com/unoplatform/uno/blob/master/build/nuget/Uno.WinUI.DevServer.nuspec#L169 (we are including *.*)